packages/graphql/src/schema-model/argument/Argument.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ValueNode } from "graphql";
+import { parseValueNode } from "../parser/parse-value-node";
+import type { AttributeType } from "../attribute/AttributeType";
+
+export class Argument {
+    // these are arguments (eg field arguments)
+    public readonly name: string;
+    public readonly type: AttributeType;
+    public readonly defaultValue?: string;
+    public readonly description: string;
```

Comment:
I don't think these properties are used in the translation layer. Are these needed in the augmented schema?

Reply:
yes